v2/interface.go

```diff
@@ -155,6 +155,11 @@ type Client interface {
 	// error. Unbind does a DELETE on the Broker's endpoint for the requested
 	// instance and binding IDs (/v2/service_instances/instance-id/service_bindings/binding-id).
 	Unbind(r *UnbindRequest) (*UnbindResponse, error)
+	// GetBinding returns configuration and credential information
```

Comment:
Add a note about this being gated on whether alpha features are enabled.

Reply:
Good idea.